region/brinstar/pink/Spore Spawn Farming Room.json

```
@@ -369,6 +369,58 @@
         "leaveWithGModeSetup": {}
       },
       "flashSuitChecked": true
+    },
+    {
+      "link": [2, 2],
+      "name": "Gain Flash Suit (Slopespark)",
+      "requires": [
+        {"or": [
+          {"canShineCharge": {"usedTiles": 20, "steepDownTiles": 2, "openEnd": 0}},
+          {"and": [
+            {"doorUnlockedAtNode": 2},
+            {"canShineCharge": {"usedTiles": 21, "steepDownTiles": 2, "openEnd": 0}}
+          ]}
+        ]},
+        {"shineChargeFrames": 150},
+        {"enemyDamage": {"enemy": "Zeb", "type": "contact", "hits": 1}},
+        "canSlopeSpark", 
+        {"shinespark": {"frames": 4, "excessFrames": 4}}
+      ],
+      "note": [
+        "After shinecharging, lure a Zeb and use it to slopespark on the first slope right of the pipe."
+      ],
+      "devNote": [
+        "No leniency is given as this is a farm room and the Zebs do little damage, even suitless."
+      ]
+    },
+    {
+      "link": [2, 2],
+      "name": "Gain Flash Suit (Slopespark) - Leave With Shinespark",
+      "requires": [ 
+        "canComplexCarryFlashSuit",
+        {"or": [
+          {"canShineCharge": {"usedTiles": 20, "steepDownTiles": 2, "openEnd": 0}},
+          {"and": [
+            {"doorUnlockedAtNode": 2},
+            {"canShineCharge": {"usedTiles": 21, "steepDownTiles": 2, "openEnd": 0}}
+          ]}
+        ]},
```

The door will have to be open for this strat
```suggestion
          {"canShineCharge": {"usedTiles": 21, "steepDownTiles": 2, "openEnd": 0}},
```